test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -178,7 +178,7 @@ public void WordBagWorkout()
                 .Read(sentimentDataPath);
 
             var est = new WordBagEstimator(Env, "text", "bag_of_words").
-                Append(new WordHashBagEstimator(Env, "text", "bag_of_wordshash"));
+                Append(new WordHashBagEstimator(Env, "text", "bag_of_wordshash", invertHash:-1));
```

Comment:
>invertHash:-1 [](start = 81, length = 13)

The default for this argument is 0  (no invert hashing) . 

On specifying `invertHash:-1`, shouldn't the output  change ?

Reply:
It got changed. You can compare bag_of_words which I'm deleting and new one which i'm putting into common folder. That header line got bigger and now includes something like this: `23602:Again, 23633:lonelyplanet` where int is hashing bucket and word is a word.

---
In reply to: [234733244](https://github.com/dotnet/machinelearning/pull/1670#discussion_r234733244) [](ancestors = 234733244)

Reply:
yeap. makes sense that we only modify the header, not the computed feature values

---
In reply to: [234733924](https://github.com/dotnet/machinelearning/pull/1670#discussion_r234733924) [](ancestors = 234733924,234733244)